py/specs/sql/uservisits.sql

```diff
@@ -1,5 +1,5 @@
 create table uservisits (
-    "sourceIP" string,
+    "sourceIP" string primary key,
```

Comment:
is `sourceIP` unique? otherwise this change will decrease the amount of records on import